marquee/marquee.ino

```diff
@@ -94,7 +94,7 @@ String CHANGE_FORM1 = "<form class='w3-container' action='/locations' method='ge
                       "<label>TimeZone DB API Key (get from <a href='https://timezonedb.com/register' target='_BLANK'>here</a>)</label>"
                       "<input class='w3-input w3-border w3-margin-bottom' type='text' name='TimeZoneDB' value='%TIMEDBKEY%' maxlength='60'>"
                       "<label>OpenWeatherMap API Key (get from <a href='https://openweathermap.org/' target='_BLANK'>here</a>)</label>"
-                      "<input class='w3-input w3-border w3-margin-bottom' type='text' name='openWeatherMapApiKey' value='%WEATHERKEY%' maxlength='60'>"
+                      "<input class='w3-input w3-border w3-margin-bottom' type='text' name='openWeatherMapApiKey' value='%WEATHERKEY%' maxlength='70'>"
```

Comment:
I didn't notice this, but does this field need to go to 70 length?

Reply:
I'm using ambient weather's api because I've got one of their weather stations, and I'm using a hacked version of your open weather map client, so I don't have to modify as much. Their API keys are a bit longer, and I didn't want to completely fork, or regularly modify that to maintain it, so I'm hoping you'll let it in, since it's just a user input length limitation. If not I understand.